docs/computing/connecting.md

```diff
@@ -70,6 +70,34 @@ ssh -X yourcscusername@puhti.csc.fi
 In `PuTTY`, X11 forwarding is enabled in the connection settings (Connection -> SSH
 -> X11: Enable X11 forwarding).
 
+## First connection
+
+When connecting for the first time, the SSH client may notify you that the host is unknown
+and ask you to confirm the connection. With the OpenSSH client, e.g., this message looks like follows:
+```
+The authenticity of host 'puhti.csc.fi' can't be established.
+ECDSA key fingerprint is SHA256:kk0Tar9opQ+6Gq0GWJdWVVvFEMeI6kW1DW1VOYveT5c.
+Are you sure you want to continue connecting (yes/no/[fingerprint])?
+```
```

Comment:
Add spaces before and after code block.
```suggestion

```
The authenticity of host 'puhti.csc.fi' can't be established.
ECDSA key fingerprint is SHA256:kk0Tar9opQ+6Gq0GWJdWVVvFEMeI6kW1DW1VOYveT5c.
Are you sure you want to continue connecting (yes/no/[fingerprint])?
```

```